pkg/vminetworkscontroller/vmi_controller_test.go

```diff
@@ -198,7 +198,7 @@ var _ = Describe("VMI IPAM controller", Serial, func() {
 			inputNADs: []*nadv1.NetworkAttachmentDefinition{
 				dummyNADWrongFormat(nadName),
 			},
-			expectedError: fmt.Errorf("failed to extract the relevant NAD information"),
+			expectedError: fmt.Errorf("failed to extract the relevant NAD information: failed to extract CNI configuration from NAD: invalid character 'h' in literal true (expecting 'r')"),
```

Comment:
nit: if we can omit please the "invalid character 'h' in literal true (expecting 'r')" would be nicer

/lgtm